test/Microsoft.AspNetCore.Mvc.Razor.Test/project.json

```diff
@@ -12,8 +12,10 @@
   ],
   "dependencies": {
     "Microsoft.AspNetCore.Http": "1.0.0-*",
+    "Microsoft.AspNetCore.Mvc": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc.DataAnnotations": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc.Formatters.Xml": "1.0.0-*",
+    "Microsoft.AspNetCore.Mvc.TagHelpers": "1.0.0-*",
```

Comment:
With @sebastienros's changes this should be pulled in by `Microsoft.AspNetCore.Mvc` now.